src/tests/multihost/alltests/conftest.py

```diff
@@ -836,13 +831,16 @@ def remove_nslcd():
 def template_sssdconf(session_multihost, request):
     """ Copy template sssd conf for multidomain tests """
     cwd = os.path.dirname(os.path.abspath(__file__))
-    remote = '/etc/sssd/sssd.conf'
+    remote = SSSD_DEFAULT_CONF
```

Comment:
Instead of assigning variable, we can directly call `SSSD_DEFAULT_CONF` on line 836.

Reply:
done